projects/elements/src/lib/lazy-elements/lazy-elements-loader.service.ts

```diff
@@ -31,14 +34,23 @@ export class LazyElementsLoaderService {
     });
   }
 
+  setGlobalOptions(options: LazyElementModuleGlobalOptions) {
```

Comment:
Yeah let's roll with the `rootOptions`, also as they are root we could inject them instead as they will not keep growing as with the previous `multi: true` configs provider